src/Server/TCPHandler.cpp

```diff
@@ -347,6 +348,7 @@ void TCPHandler::runImpl()
          */
         std::unique_ptr<DB::Exception> exception;
         bool network_error = false;
+        bool close_connection = false;
```

Comment:
As far as I can see `close_connection` is not used anywhere else so it could be just `user_expired` for better readability